generate_brick.py

```diff
@@ -687,6 +688,7 @@ def add_definitions():
 define_classes(equipment_subclasses, BRICK.Equipment)
 define_classes(collection_classes, BRICK.Collection)
 define_classes(hvac_subclasses, BRICK.HVAC_Equipment)
+define_classes(hvac_valve_subclasses, BRICK.HVAC_Valve)
 define_classes(valve_subclasses, BRICK.Valve)
```

Comment:
Now that `Valve` is the top level class in the `valve_subclasses` dictionary, this line should be:

```python
define_classes(valve_subclasses, BRICK.HVAC_Equipment)
```

otherwise there is a loop where `Valve` is a subclass of `Valve`